### PR TITLE
Update theme.view.php issue Make Theme UI clearer #19

### PR DESCRIPTION
**Pull request by siramsay**

### CODE REVIEW

File: deploy/src/perihelion/php/view/theme.view.php [hunk 1]
```diff
@@ -120,10 +120,10 @@ public function themeList() {
 													$h .= "<td class=\"text-center\">" . $theme->themeCreationDateTime . "</td>";
 													$h .= "<td class=\"text-center\">";
 														if ($themeID == $site->themeID) {
-															$h .= "<span class=\"fas fa-check\" style=\"color:#000;\"></span>";
+															$h .= "<span class=\"fas fa-check\" style=\"color:#000;\"></span> Current";
 														} else {
 															$select_theme = '/' . Lang::languageUrlPrefix() . 'designer/themes/select/' . $themeID . '/';
-															$h .= "<a class=\"btn btn-secondary btn-sm\" href=\"" . $select_theme . "\"><span class=\"fas fa-check\" style=\"color:#fff;\"></span></a>";
+															$h .= "<a class=\"btn btn-secondary btn-sm\" href=\"" . $select_theme . "\"><span class=\"\" style=\"color:#fff;\"></span> Select</a>";
```
Comment (chishiki) [inline deploy/src/perihelion/php/view/theme.view.php:126]: this change is fine thanks for plugging away at it

just one thing when we add text we want to remember to use `Lang::getLang('xxxxxxx')`

Comment (siramsay) [reply deploy/src/perihelion/php/view/theme.view.php:126]: nice, I'll add the Lang::getLang('xxxxxxx')

Comment (chishiki) [reply deploy/src/perihelion/php/view/theme.view.php:126]: thanks!

**Review by chishiki — CHANGES_REQUESTED** [review]
lemme know when you've localized those "select" and i'll push this in thanks!

File: deploy/src/perihelion/php/view/theme.view.php (hunk 1 above)
Comment (siramsay) [reply deploy/src/perihelion/php/view/theme.view.php]: just added that and this says outdated now. unsure if you can see the new changes?

**Comment by siramsay** [comment]
It says added 2 commits above so I guess they are added to the pull request, please let me if this is done right. Thanks

**Pull request merged**